### ISSUE TITLE
redash-migrate: admin and default groups should not be duplicated

### ISSUE THREAD

**Issue by susodapop** [getredash/redash-toolbelt#74]
Per [this comment](https://github.com/getredash/redash-toolbelt/discussions/68#discussioncomment-1468728). 

When moving the ORIGIN `admin` and `default` groups, we can just copy users into the existing groups on the DESTINATION.

**Comment by wtfzambo** [comment]
It seems that the reason the duplication happens is due to [this](https://github.com/getredash/redash-toolbelt/blob/29ff3b69d7204d4b9481ca6119003a2f49334674/redash_toolbelt/examples/migrate.py#L186) line:

```python
if id in meta["groups"]:
            print(f"Origin group {id} -- SKIP -- Already exists in destination")
            continue
```

However, when I check `app.redash.io/mycompany/api/groups` w.r.t. a self hosted solution, the IDs don't match. See screenshots from my two accounts. First ones are Redash, second ones are self-hosted:

## Redash
![image](https://user-images.githubusercontent.com/47462792/137363911-65c19c40-ddbf-4aee-a1a6-f9c319f1152d.png)
![image](https://user-images.githubusercontent.com/47462792/137363943-27a9b556-6dff-405f-867a-96f421fe4725.png)

## Self-hosted
![image](https://user-images.githubusercontent.com/47462792/137364018-7c3bb9a8-7bfe-45bf-8bd4-8489449dc784.png)
![image](https://user-images.githubusercontent.com/47462792/137365058-28bc00ca-cebd-43f2-a391-ab597c2621de.png)

---

One possible solution would be to check the origin and if it contains `app.redash.io`, make the comparison based on names instead of IDs, although it's a bit redundant since I suppose most people are going to migrate from Redash to self-hosted, instead of self-hosted to self-hosted.

e.g.:

```python
def import_groups(orig_client, dest_client):

    o_groups = orig_client._get("api/groups").json()

    for group in o_groups:

        id = group["id"]
        name = group["name"]
        origin_url = meta["settings"]["origin_url"]

        if 'app.redash.io' in origin_url and name in meta["group_names"]:
            print(f"Origin name {name} -- SKIP -- Already exists in destination")
            continue

        elif id in meta["groups"]:
            print(f"Origin group {id} -- SKIP -- Already exists in destination")
            continue


        # Post to GroupListResource to create the group in the destination
        resp = dest_client._post("api/groups", json={"name": name}).json()
etc...
```

`meta["group_names"]` would need to be added to the global `meta` dict, however I am a bit confused because I don't see where exactly destination data is fetched.

Alternatively, the check could be made solely on group_names, skipping the `if 'app.redash.io' in origin_url` condition entirely.

**Comment by susodapop** [comment]
Closed by #82
